src/instance.rs

```diff
@@ -234,7 +234,7 @@ pub async fn play(path: PathBuf, executable: PathBuf, name: String) -> Result<()
     err_path.push(format!("{}.err", time));
     let mut err = File::create(err_path)?;
 
-    info!("Launching {}", name);
+    info!("Launching {} via executable {}", name, executable.to_string_lossy());
```

Comment:
On the long run, we might want to make this path relative to the instance dir, so it doesn't become too long.

Reply:
Completely agree. I do not understand the code well enough to be sure about where the current working directory is and if you change it during processing, therefore I used absolute paths everywhere.